README.md

```diff
@@ -23,13 +23,11 @@ Deploying a Python application couldn't be easier:
     $ git push heroku master
     ...
     -----> Python app detected
-    -----> Installing python-3.6.2
-         $ pip install -r requirements.txt
-           Collecting requests (from -r requirements.txt (line 1))
-             Downloading requests-2.12.4-py2.py3-none-any.whl (576KB)
-           Installing collected packages: requests
-           Successfully installed requests-2.12.4
-
+   -----> Installing python-3.6.2
+   -----> Installing pip
+   -----> Installing requirements with latest pipenv...
+          ...
+          Installing dependencies from Pipfile...
     -----> Discovering process types
```

Comment:
The indentation is one off here compared to the existing lines - not sure which of the new or old lines are correct?

Reply:
nice catch